packages/graph/hash_graph/lib/graph/src/store/postgres/mod.rs

```diff
@@ -583,11 +587,13 @@ where
 
         Ok((
             version_id,
-            PersistedOntologyMetadata::new(
-                PersistedOntologyIdentifier::new(uri, owned_by_id),
-                created_by_id,
-                UpdatedById::new(created_by_id.as_account_id()),
-                None,
+            OntologyElementMetadata::new(
+                uri.into(),
```

Comment:
Maybe we want to continue using `VersionedUri` to avoid all of the `into` calls.. 

I think there is some benefit to having a type we own though. It lets us have a different interface (`baseId` which is consistent with the `EntityEditionId` and will play nicer in subgraphs and such, and we could potentially even impl a generic trait).

Reply:
I think this is fine for now. Even though the conversion is annoying and the required clones are suboptimal it's more clear.

We spoke about adding `repr(C)`, but I think it's not worth it because (a) we introduce `unsafe` to this code base and (b) we would require to keep the type system package and the Graph crate in sync. While I don't expect that the `VersionedUri` to change it's a risk that we forgot to update one structure.